standard/variables.lua

```diff
@@ -14,6 +14,11 @@ function Variables.varDefine(name, val)
 	return mw.ext.VariablesLua.vardefine(name, val)
 end
 
+function Variables.varDefineEcho(name, val)
+	mw.ext.VariablesLua.vardefine(name, val)
+	return val
```

Comment:
```suggestion
	return mw.ext.VariablesLua.vardefineecho(name, val)
```
(untested)

Reply:
yeah it works but the behavior is slightly different

we should decide which we want

with my version if you give it a number it will return a number
with your suggestion it returns the number as a string

btw. for fun i did some (crazy) performance test and the version i have is minimal better
let it write 500k vars per test run and mine was in average (with 10 test runs) ~0.05s faster

Reply:
It should return the same type as a call to `varDefault` would do

Reply:
so string it is